x-pack/plugins/observability/server/types/rest_specs/slo.ts

```diff
@@ -86,11 +94,33 @@ const updateSLOResponseSchema = t.type({
   updated_at: dateType,
 });
 
+const findSLOResponseSchema = t.type({
+  page: t.number,
+  per_page: t.number,
+  total: t.number,
+  results: t.array(
+    t.type({
+      id: t.string,
```

Comment:
I was checking to see if we have this type already (like SLO Response Schema) and I saw `getSLOResponseSchema` which has extra information such as `summary`, what is the reason for removing `summary` from the response and does it make sense to have a dedicate schema for the returned SLO from different APIs?

Reply:
Good question. `summary` requires an extra ES query per slo in order to get the SLI Data, and compute metrics like error budget consumption. I did not want to bloat this request with many ES queries (one per SLO) at the moment. 

That being said, I think in the future we could definitely add the summary in the response (and I think it's useful to see the current sli value & error budget of every SLO listed on the page). This would require to implement a more optimal way of fetching the SLI value for a group of SLO, so we would make only 1 request to ES instead of 1 per SLO. This should not be too complex: adding a terms aggregation on the slo.id with a filter. 

But for the main purpose of this endpoint at the moment: listing SLO for the burn rate rule editor, this summary data is not useful.

Regarding having dedicated schema definitions for the routes/application services, I think it's easier to evolve them independently without dealing with some common parts.